### PR TITLE
ArrayList and Junit-test checklists corrected

### PR DESCRIPTION
**Pull request by Rommelua**

### CODE REVIEW

File: java-core/junit/registration-validation.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
-### Common mistakes
+## Common mistakes (jv-registration-validation)
 
 * Don’t ignore checkstyle rules.
 * Figure out which way is better to initialize your RegistrationServiceImpl class instance: `@BeforeEach` or `@BeforeAll`, what’s the difference?
 * Make sure you name your methods according to [convention](https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names).
 * Add tests for all possible User's parameters (null login/password/age, user under 18/18/over 18 years old, negative age, and so on...).
+* In your testы very important to check **all possible edge cases**. For example, we have min password length 6 characters.
```
Comment (hronograf) [inline java-core/junit/registration-validation.md:7]:
```suggestion
* In your tests very important to check **all possible edge cases**. For example, we have min password length 6 characters. 
```
